source/ARTICLE.md

```diff
@@ -1,134 +1,94 @@
+### Software Development
+- Explain which tools/hardware are employed in the project.
+- Reference the different issues which were created to address the objectives of the project.
+- Explain which testing procedures have been implemented.
+
+### Guidelines/Workflow or educational resources/event
+- Specify the target community for the resources.
+- Specify how you assessed the utility of proposed resources for the target community.
+- Describe the support resources for the proposed resources (documentation, website)
+- Describe the plan for dissemination of the resources in the target community.
+- Describe the plan for measuring the engagement and effectiveness of the resources in the target community.
+
+### Data analysis
+- Explain the study design including the groups and measures.
+- Please describe the process by which you will collect your data.
 - Justify and identify the sample size of your study. How many units will be analyzed in the study?
-- Describe each variable that you will measure. 
+- Describe each variable that you will measure.
 - Please list all the basic steps of your preprocessing.
 - Please briefly explain how the preprocessing of the data will be held? (methods, tools, pipeline etc.)
-- Please briefly explain how you will analyze the data. 
+- Please briefly explain how you will analyze the data.
 - What statistical model will you use to test each hypothesis?
-- How will you determine what data or samples, if any, to exclude from your analyses? How will outliers be handled? Will you use any awareness check? Is there a minimum number of trials participants should contribute to the analysis?
+- How will you determine what data or samples, if any, to exclude from your analyses?
 - How will you deal with incomplete or missing data?
 
-## Progress (required)
-- The process completed before Brainhack
-Explain the work has been completed before the hacking.
-
-- The process completed throughout Brainhack 
-Explain the work has been completed during the hacking period.
-
-- The process completed after Brainhack
-Explain the work has been completed during the hacking period.
+### Educational resources or event
+- Specify the target community for the educational resource/event.
+- Specify how you assessed the utility of proposed educational resources/event for the target community.
+- Describe the support resources for the proposed educational resources/event (documentation, website)
+- Describe the plan for dissemination of the educational resources/event in the target community.
+- Describe the plan for measuring the engagement and effectiveness of the educational resources/event in the target community.
```

Comment:
These lines repeat the similar if not the same content from lines 43-48. We should keep on or the other.